### PR TITLE
Correction on the consistency of design and buttons #13958

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Improves the consistency of the design, and the color of the buttons and their state when you hover the mouse

### Why is it needed?
Change the colors of the buttons

After fix:
After

### Related issue(s)/PR(s)
Issue #13958

### CODE REVIEW

File: _build/templates/default/sass/_colors-and-vars.scss [hunk 1]
```diff
@@ -88,19 +89,14 @@ $buttonColorPrimary: $white;
 $buttonColorPrimaryHover: $white;
 $buttonColorPrimaryActive: $white;
 $buttonColorPrimaryDisabled: $white;
-$buttonPrimaryBg1: $primary1;
-$buttonPrimaryBg2: $primary2;
-$buttonPrimaryBg1Hover: darken($buttonPrimaryBg1, 6%);
-$buttonPrimaryBg2Hover: darken($buttonPrimaryBg2, 6%);
-$buttonPrimaryBg1Active: darken($buttonPrimaryBg1, 12%);
-$buttonPrimaryBg2Active: darken($buttonPrimaryBg2, 12%);
+$buttonPrimaryBgActive: darken($green, 12%);
```
Comment (alroniks) [inline _build/templates/default/sass/_colors-and-vars.scss]: Make sense to use `$darkGreen` variable here.

File: _build/templates/default/sass/_colors-and-vars.scss [hunk 2]
```diff
@@ -20,6 +20,7 @@ $yellow: #FFEA80;
 $orange: #F49548;
 $red: #BE0000;
 $green: #6CB24A;
+$darkGreen: darken($green, 12%);
```
Comment (alroniks) [inline _build/templates/default/sass/_colors-and-vars.scss]:
```suggestion
$greener: darken($green, 12%);
```

**Review by alroniks — COMMENTED** [review]
My questions are not strictly required, just some notices.

File: _build/templates/default/sass/_colors-and-vars.scss (hunk 1 above)
Comment (Ibochkarev) [reply _build/templates/default/sass/_colors-and-vars.scss]: Thanks

**Review by Mark-H — DISMISSED** [review]

**Comment by Ibochkarev** [comment]
@JoshuaLuckers @Jako Please check this PR

**Comment by JoshuaLuckers** [comment]
Thanks for notifying. I will check the PR when I have time, in the meantime would you mind fixing the conflict?

**Comment by Ibochkarev** [comment]
@JoshuaLuckers Conflicts fix.

**Comment by OptimusCrime** [comment]
Why was this closed?

**Comment by Ibochkarev** [comment]
@OptimusCrime I was wrong in working with git. I will fix some points and today I will open a new pull request

**Pull request closed without merging**